### ISSUE TITLE
Remove .DS_Store files

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass-website#4]
Probably to be removed and blacklisted in .gitignore?

```
find . -name .DS_Store
./static/images/.DS_Store
./static/.DS_Store
./content/.DS_Store
./content/download/.DS_Store
./content/about/.DS_Store
./.DS_Store
./themes/grass/static/plugins/fontawesome/.DS_Store
./themes/grass/static/plugins/.DS_Store
./themes/grass/static/plugins/themify-icons/.DS_Store
./themes/grass/static/.DS_Store
./themes/grass/images/.DS_Store
./themes/grass/assets/.DS_Store
./themes/grass/layouts/.DS_Store
./themes/grass/layouts/download/.DS_Store
./themes/grass/layouts/about/.DS_Store
```

**Comment by landam** [comment]
See related #5

**Comment by neteler** [comment]
Fixed in #5
